src/content/api/tokens.mdx

```diff
@@ -98,6 +98,11 @@ A [Redemption Condition](#redemption-condition-model) is created for each [Merch
   <Property name="duration" type="string">
     Number of `period` until token expiration.
   </Property>
+
+  <Property name="ceil" type="string">
+    Interval to round token expiry up to.
+    Supported values are `hour`, `day`, `week`, `month` and `year`.
+  </Property>
```

Comment:
I think this field should also have an optional flag on it.

Reply:
There are no optional flags that we use in docs but I can add required to the other two properties